snuba/datasets/eventsv2.py

```diff
@@ -0,0 +1,75 @@
+from datetime import timedelta
+from typing import Sequence
+
+from snuba.datasets import TimeSeriesDataset
+from snuba.datasets.dataset_schemas import DatasetSchemas
+from snuba.datasets.factory import get_dataset
+from snuba.datasets.schemas.join import (
+    JoinConditionExpression,
+    JoinCondition,
+    JoinedSchema,
+    JoinStructure,
+    JoinType,
+    SchemaJoinedSource,
+    SubJoinSource,
+)
+from snuba.query.extensions import PERFORMANCE_EXTENSION_SCHEMA, PROJECT_EXTENSION_SCHEMA
+from snuba.query.schema import GENERIC_QUERY_SCHEMA
+from snuba.request import RequestSchema
+from snuba.schemas import get_time_series_extension_properties
+
+
+class EventsV2(TimeSeriesDataset):
```

Comment:
This is kind of experimental for now, not sure if the composite dataset we will use in production will be called eventsV2

Reply:
Should we be concerned about reusing terms between sentry and snuba? `eventsv2` in sentry is ~= to discover2.

Reply:
discoverv1.5 ?